docs/src/main/paradox/discovery.md

```diff
@@ -521,6 +569,53 @@ Notes:
 
 * If Akka management port tag is not found on service in Consul the implementation defaults to catalog service port.
 
+## Discovery Method: Aggregate multiple discovery mechanisms
+
+Aggregate discovery allows multiple discovery mechanisms to be aggregated e.g. try and resolve
+via DNS and fall back to configuration.
+
+To use aggregate discovery add its dependency as well as all of the discovery mechanisms that you 
+want to aggregate.
+
+@@dependency[sbt,Gradle,Maven] {
+  group="com.lightbend.akka.discovery"
+  artifact="akka-discovery-aggregate_2.12"
+  version="$version$"
+}
+
+Configure `akka-aggregate` as `akka.discovery.method` and which discovery mechanisms are tried and in which order.
+
+```
+akka {
+  discovery {
+    method = akka-aggregate
```

Comment:
not sure why "akka"

Reply:
All of the inbuilt ones have the prefix akka-dns, akka-config etc

Reply:
That is the complaint ;) about both akka-dns and akka-config actually hm... akka-dns i could see at least a bit since it is "the akka impl" rather than some other one, but this is not "akka config"

Reply:
I agree that it would read better without the prefix.